VideoApi/VideoApi.DAL/Queries/GetConferenceInterpreterRoomsByDateQuery.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+using VideoApi.Services.Kinly;
+
+namespace VideoApi.DAL.Queries
+{
+    public class GetConferenceInterpreterRoomsByDateQuery : IQuery
+    {
+        public DateTime DateStamp  { get; set; }
+
+        public GetConferenceInterpreterRoomsByDateQuery(DateTime dateStamp)
+        {
+            DateStamp = dateStamp;
+        }
+    }
+    
+    public class GetConferenceInterpreterRoomsByDateQueryHandler : IQueryHandler<GetConferenceInterpreterRoomsByDateQuery, List<HearingAudioRoom>>
+    {
+        private readonly VideoApiDbContext _context;
+
+        public GetConferenceInterpreterRoomsByDateQueryHandler(VideoApiDbContext context)
+        {
+            _context = context;
+        }
+        
+        public Task<List<HearingAudioRoom>> Handle(GetConferenceInterpreterRoomsByDateQuery query)
+        {
+            var results = from conf in _context.Conferences
+                          join even in _context.Events on conf.Id equals even.ConferenceId
```

Comment:
Any reason to shorten these or could we change these to conference and event to be more readable?

Reply:
intellisense was not allowing me to name event as it is reserved keyword I think.

Reply:
Fixed.. thanks